### ISSUE TITLE
renaming from context of custom renderer is not working

### ISSUE THREAD

**Issue by Nishchit14** [lukasbach/react-complex-tree#83]
**Describe the bug**
I am using a custom renderer for items and It has a menu dropdown, from where the user can click on the rename option to rename the item, but the `context.startRenaming` method doesn't work at all.

here is the renderer 

```ts
renderItem: ({ item, depth, children, title, context, arrow }) => {

    let _startRenaming = ()=> {

      console.log(context, "... _startRenaming")
      context.startRenamingItem(); // <= this method is empty and make no effect
    }
}
```

also, I see that this function is not implemented yet, this might be the reason https://github.com/lukasbach/react-complex-tree/blob/665fce7c44886b59cf7fb3a4d17bd66790a72b26/packages/core/src/treeItem/useTreeItemRenderContext.ts#L94

**To Reproduce**
1. try to rename from the context of `renderItem` renderer

**Expected behavior**
The item should be renamed programmatically from the `context` 

**Desktop (please complete the following information):**

- OS: macOS
- Browser chrome, safari
- Version 1.1.8

**Comment by lukasbach** [comment]
Hi @Nishchit14, thank you for your report! This is fixed in version 1.1.9.

**Comment by Nishchit14** [comment]
It's not working on `1.1.11`. `context.startRenamingItem(); ` is fires `onRenamingItem` callback but the UI has no change.

**Comment by lukasbach** [comment]
After investigating this again, I'm not even sure I properly fixed it the last time 🙃 

This is now actually fixed as of 2.0.0. Storybook example: https://rct.lukasbach.com/storybook/?path=/story/core-basic-examples--tree-item-context-renaming
